GocadRelatedScripts/createFaultFromCurve.py

```diff
@@ -1,5 +1,5 @@
 import numpy as np
-from math import tan, pi, pow, cos, sin
+from math import tan, pi, pow, cos, sin, atan2
```

Comment:
We could only use numpy, couldnt we?

Reply:
numpy functions are much slower, because they expect large arrays.
Here for instance, I loop over nx and by, this is not parallelized:

```
    for i in range(0, nx):
        for j in range(1, nd):
            if args.dipType == 0:
                mydip = dip
            elif args.dipType == 1:
                mydip = dipangle(depth[j] + vertices[i, 0, 2])
            else:
                mydip = aDip[i]
            ud = -(1.0 / tan(mydip) * av1[i, :] - uz)
```